### ISSUE TITLE
hyperparameters for pre-training on TieredImageNet?

### ISSUE THREAD

**Issue by Yzy5020** [icoz69/DeepEMD#38]
What are the best hyperparameters for pre-training on TieredImageNet?

Should I keep the hyperparameters in the train_pretrain.py unchanged？

**Comment by icoz69** [comment]
hi generally the parameters to adjust are lr scheduler, such as decay step and decay strength. I have recently found that using cosine lr scheduler without annealing is good as no hyper-parameters about lr scheduler is needed any more, and you only need to find a good start lr.
